src/Components/ExtractLevelsMenu.jsx

```diff
@@ -105,28 +106,36 @@ export default function ExtractLevelsMenu({listOfOptions, icon, title}) {
     }
   }, [model])
 
-  let sampleHeights = []
-
+  const showExtractMenu = async () => {
+    const allStor = await extractHeight(model)
+    console.log(allStor)
+    let sampleHeights = []
 
-  // PLACEHOLDER VALUES
-  /* eslint-disable */
-  const EISVOGEL = [
-    -2.24, 0, 3.5,
-    6.4, 9.5, 12.6,
-    15.7, 18.8, 21.81,
-    27.12,
-  ]
+    // PLACEHOLDER VALUES
+    /* eslint-disable */
+    const EISVOGEL = [
+      -2.24, 0, 3.5,
+      6.4, 9.5, 12.6,
+      15.7, 18.8, 21.81,
+      27.12,
+    ]
+  
+    /* eslint-enable */
+    // sampleHeights = EISVOGEL
+    sampleHeights = allStor
+    const sampleHeightsIndex = []
+    for (let i = 0; i < sampleHeights.length; i++) {
+      sampleHeightsIndex[i] = i
+    }
 
-  const sampleHeightsIndex = [0, 1, 2, 3, 4, 5, 6]
-  /* eslint-enable */
+    const planeoffset = 0.5
+    sampleHeightsIndex.forEach((data) => {
+      floorplanMenu.push(
+          <MenuItem onClick={() => createFloorplanPlane(sampleHeights[data], sampleHeights[data + 1] - planeoffset)}>  L{data} </MenuItem>)
```

Comment:
@pablo-mayrgundter, I'm trying to get the generated submenu to show up on the UI after the promise is fulfilled